guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
+The inherited value is based on the `host_registration_remote_execution` parameter.
+It can be inherited e.g. from host group, operating system, organization.
+When overridden, the selected value will be stored on host parameter level.
 
 . From the *Setup Insights* list, select whether you want to install `insights-client` and register the hosts to Insights.
 +
-The Insights tool is available for Red Hat Enterprise Linux only. It has no effect on other operating systems.
+The Insights tool is available for {RHEL} only. It has no effect on other operating systems.
++
 You must enable the following repositories on a registered machine:
 
 * RHEL 6: `rhel-6-server-rpms`
 * RHEL 7: `rhel-7-server-rpms`
-* RHEL 8: `rhel-8-for-x86_64-appstream-rpms` (The `insights-client` package is installed by default on RHEL 8 except in environments whereby RHEL 8 was deployed with "Minimal Install" option)
+* RHEL 8: `rhel-8-for-x86_64-appstream-rpms`
 +
+The `insights-client` package is installed by default on RHEL 8 except in environments whereby RHEL 8 was deployed with "Minimal Install" option.
 
-. Optional: {Project} uses the JSON Web Tokens (JWT) for authentication.
+. Optional: In the *Token lifetime (hours)* field, change the validity duration of the JSON Web Token (JWT) that {Project} uses for authentication.
 The duration of this token defines how long the generated `curl` command works.
-If you want to change the duration of the token, enter the required duration to the *Token lifetime (hours)* field. You can set the duration to 0 - 999 999 hours or unlimited.
+You can set the duration to 0 - 999 999 hours or unlimited.
 +
-Note that {Project} uses the permissions of the user who generates the `curl` command for authorization of hosts.
+Note that {Project} applies the permissions of the user who generates the `curl` command to authorization of hosts.
 If the user loses or gains additional permissions, the permissions of the JWT change too.
 Therefore, do not delete, block, or change permissions of the user during the token duration.
-Also, the scope of JWT is limited to the registration endpoints only and cannot be used anywhere else.
-. Optional: In the *Remote Execution Interface* field, enter a network interface that hosts must use for the SSH connection.
++
+The scope of the JWTs is limited to the registration endpoints only and cannot be used anywhere else.
+. Optional: In the *Remote Execution Interface* field, enter the identifier of a network interface that hosts must use for the SSH connection.
 If you keep this field blank, {Project} uses the default network interface.
-. Optional: *Install packages* - Install packages on the host when registered. Can be set by `host_packages` parameter
+. Optional: In the *Install packages* field, list the packages (separated with spaces) that you want to install on the host upon registration. This can be set by the `host_packages` parameter.
```

Comment:
```suggestion
. Optional: In the *Install packages* field, list the packages (separated with spaces) that you want to install on the host upon registration.
This can be set by the `host_packages` parameter.
```